libretroshare/src/pqi/pqissl.cc

```diff
@@ -1253,8 +1262,10 @@ int pqissl::accept_locked( SSL *ssl, int fd,
 	if (rsPeers->servicePermissionFlags(PeerId()) & RS_NODE_PERM_REQUIRE_WL)
 		checking_flags |= RSBANLIST_CHECKING_FLAGS_WHITELIST;
 
-	if(rsBanList && !rsBanList->isAddressAccepted(
-	            foreign_addr, checking_flags, check_result ))
+    if( RsX509Cert::getCertSslId(*SSL_get_peer_certificate(ssl)) != PeerId())
+        std::cerr << "(EE) pqissl::accept_locked(): PeerId() is " << PeerId() << " but certificate ID is " << RsX509Cert::getCertSslId(*SSL_get_peer_certificate(ssl)) << std::endl;
+
+	if(rsBanList && !rsBanList->isAddressAccepted( foreign_addr, checking_flags, check_result ))
 	{
 		RsErr() << __PRETTY_FUNCTION__
 		        << " Refusing incoming SSL connection from blacklisted "
```

Comment:
This should be RsInfo() instead of RsErr()

Reply:
done